src/js/svgDrawingArea.js

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // A base SVG drawing area component
+    // Creates a basic SVG drawing area for use by implementing grades that actually
+    // draw charts; includes hooks for accessible title and description in model
+
+    fluid.defaults("floe.svgDrawingArea", {
+        gradeNames: ["floe.d3ViewComponent", "autoInit"],
```

Comment:
i suspect the answer is that wherever some of this structure was originally copy-pasted from was from the dawn of time. :)